examples/north_sea/spinup.py

```diff
@@ -0,0 +1,49 @@
+from thetis import *
+import time as time_mod
+from model_config import *
+import argparse
+import os
+import sys
+
+
+# Parse user input
+parser = argparse.ArgumentParser(
+    description="North Sea spin up",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+parser.add_argument("--no-exports", action="store_true")
+args = parser.parse_args()
+no_exports = args.no_exports
+
+# Setup solver
+solver_obj, start_time, update_forcings = construct_solver(
+    output_directory="outputs_spinup",
+    spinup=True,
+    store_station_time_series=not no_exports,
+    no_exports=no_exports,
+    start_date=datetime.datetime(2022, 1, 1, tzinfo=sim_tz),
+    end_date=datetime.datetime(2022, 1, 15, tzinfo=sim_tz),
+)
+output_dir = solver_obj.options.output_directory
+mesh2d = solver_obj.mesh2d
+elev_init = Function(solver_obj.function_spaces.P1_2d)
+print_output(f"Exporting to {output_dir}")
+solver_obj.assign_initial_conditions(elev=elev_init)
+update_forcings(0.0)
+
+# Time integrate
+tic = time_mod.perf_counter()
+solver_obj.iterate(update_forcings=update_forcings)
+toc = time_mod.perf_counter()
+print_output(f"Total duration: {toc-tic:.2f} seconds")
+if os.getenv("THETIS_REGRESSION_TEST") is not None:
+    sys.exit(0)
+
+# Store spun-up state
+h5_dir = create_directory(os.path.join(output_dir, "hdf5"))
+exporter.HDF5Exporter(solver_obj.function_spaces.U_2d, h5_dir, "Velocity2d").export(
+    solver_obj.fields.uv_2d
+)
+exporter.HDF5Exporter(solver_obj.function_spaces.H_2d, h5_dir, "Elevation2d").export(
+    solver_obj.fields.elev_2d
+)
```

Comment:
I would prefer to use normal exports to create the spun-up state, e.g.
```
fields_to_export = []
fields_to_export_hdf5 = ['elev_2d', 'uv_2d']
export_time = 24 * 3600.
```
Then to start the actual run you'd need to do `load_state(15, ..., t=0, iteration=0)` to reset the clock.
`HDF5Exporter` objects should only be used internally, so I would not include their usage in an example.